<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MnemonicDialog</class>
 <widget class="QDialog" name="MnemonicDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mnemonic generation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Your wallet seed:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="mnemonicEdit">
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="descriptionLabel">
     <property name="text">
      <string>Please save these words on paper (order is important). This seed will allow you to recover your wallet.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="generateButton">
       <property name="text">
        <string>Generate another seed</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MnemonicDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MnemonicDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mnemonicEdit</sender>
   <signal>textChanged()</signal>
   <receiver>MnemonicDialog</receiver>
   <slot>mnemonicChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>generateButton</sender>
   <signal>clicked()</signal>
   <receiver>MnemonicDialog</receiver>
   <slot>generateMnemonic()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>mnemonicChanged()</slot>
  <slot>generateMnemonic()</slot>
 </slots>
</ui>
